app/src/main/java/fr/neamar/kiss/result/ContactsResult.java

```diff
@@ -283,8 +283,11 @@ public View inflateFavorite(@NonNull Context context, @NonNull ViewGroup parent)
     private void launchContactView(Context context, View v) {
         Intent viewContact = new Intent(Intent.ACTION_VIEW);
 
-        viewContact.setData(Uri.withAppendedPath(ContactsContract.Contacts.CONTENT_LOOKUP_URI,
-                String.valueOf(pojo.lookupKey)));
+        Uri contactUri = ContactsContract.Contacts.CONTENT_LOOKUP_URI;
+        contactUri = Uri.withAppendedPath(contactUri, String.valueOf(pojo.lookupKey));
+        contactUri = Uri.withAppendedPath(contactUri, String.valueOf(pojo.getContactId()));
+
+        viewContact.setData(contactUri);
```

Comment:
There was is similar code in place in this class for ```contactIcon.assignContactUri``` too, can you please change it there too?

Else tap on icon would behave a little different than click on other part of result.